### DISCUSSION TITLE
Can't find example of how to use this with Laravel Livewire will be great.Please give me an example, thanks =)

### DISCUSSION THREAD

**Discussion by vadykoo** [spatie/laravel-honeypot#70] [Q&A]
Please give me an example, I can't find

> Currently don't have the time to add an example for this. Feel free to PR one to the docs.

_Originally posted by @vadykoo in https://github.com/spatie/laravel-honeypot/issue_comments/707789479_

**Comment by masterix21** [comment]
Take a look here: [#107](https://github.com/spatie/laravel-honeypot/pull/107)

I hope it will be merged :)
